### ISSUE TITLE
Add jskos-server at services

### ISSUE THREAD

**Issue by nichtich** [gbv/coli-conc.gbv.de#31]
jskos-server can be used for other applications as well, so list it at https://coli-conc.gbv.de/services/. May wait until jskos-server can more easily be used as web application (https://github.com/gbv/jskos-server/issues/106)

**Comment by stefandesu** [comment]
Yeah, currently it is only mentioned via [Concordance Registry](https://coli-conc.gbv.de/concordances/) and Mappings API. Maybe under APIs we should have "KOS and Concordance API" or something.
